paddle/fluid/operators/detection/distribute_fpn_proposals_op.cu

```diff
@@ -26,7 +26,7 @@ namespace operators {
 using Tensor = framework::Tensor;
 using LoDTensor = framework::LoDTensor;
 
-static constexpr int kNumCUDAThreads = 512;
+static constexpr int kNumCUDAThreads = sizeof(uint64_t) * 8;
```

Comment:
Why change to `sizeof(uint64_t) * 8` ?

Reply:
It will case cudalasterror when kNumCUDAThreads=512

Reply:
It will cause CudaLastError when kNumCUDAThreads = 512;